README.md

```diff
@@ -0,0 +1,28 @@
+<h4> 1. The purpose of clisnt-libs-java-samples </h4>
+  
+All examples shown below fails, because samples does not contain appropriate adscore signature
```

Comment:
lets add info that user need to fill this data with his own, ideally it should be a properties file ?

Reply:
Added some info on how and to which example user needs to provide appropriate arguments.

For now, passing arguments to samples bases on command-line args:
e. g.
```
java -jar build/libs/client-libs-java-samples-1.0.4.jar all-args-sample {arg2} ... {argn...}
```
I believe is a bit simpler than using a properties file, but we can switch to use a properties file if necessary.